requirements/requirements.in

```diff
@@ -30,6 +30,7 @@ tox
 flake8
 coverage
 django-debug-toolbar
+httmock
```

Comment:
I'd like to avoid installing tools like this on our production servers. How difficult would it be to create a requirements.test.txt

Reply:
@rnagle or @ekivemark any idea if splitting these out would cause problems or require changes in our CI or build pipelines?

Reply:
@whytheplatypus I think all you'll need to do is update [tox.ini](https://github.com/CMSgov/bluebutton-web-server/blob/develop/tox.ini#L9) to use `requirements.test.txt`.